docs/log_collection.md

```diff
@@ -243,3 +243,4 @@ If your existing codebase is using Timber, you can forward all those logs to  Da
 [3]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
 [4]: https://docs.datadoghq.com/logs/processing/attributes_naming_convention/
 [5]: https://docs.datadoghq.com/tagging/
+[6]" https://docs.datadoghq.com/real_user_monitoring/installation/?tab=us
```

Comment:
```suggestion
[6]: https://docs.datadoghq.com/real_user_monitoring/installation/?tab=us
```

Reply:
This link is redirecting to https://docs.datadoghq.com/real_user_monitoring/ Did you mean to go here maybe? https://docs.datadoghq.com/real_user_monitoring/browser/?

Reply:
Actually it would be more relevant to link to this page: https://docs.datadoghq.com/real_user_monitoring/android/?tab=us

Reply:
Ya I actually meant to include https://docs.datadoghq.com/real_user_monitoring/android/?tab=us to get users a better idea on where to go to generate the application_id